File: Problems/PaintCalculator_9/PaintCalculator.py
import math

feetPerGallon = 350;
while True:
  length = input("What's the length of the room? ")
  width = input("What's the width of the room? ")
  if length.isnumeric() and width.isnumeric():
    break
  else:
    print("Please enter two positive numbers")

lengthInt = float(length)
widthInt = float(width)

area = lengthInt * widthInt
gallons = math.ceil(area / feetPerGallon)

print("You will need to purchase %f gallons of paint to cover %f square feets" % (gallons, area))

File: Problems/PaintCalculator_9/PaintCalculator.cpp
#include <iostream>
#include <string>
#include <stdlib.h>
#include <math.h>

using namespace std;

int main()
{
    const double feetPerGallons = 350;

	string length, width;
	int gallons;
	double lengthDbl, widthDbl, squareArea;

	cout << "What's the length of the room? ";
	cin >> length;
	cout << "What's the width of the room? ";
	cin >> width;

	lengthDbl = atof(length.c_str());
	widthDbl = atof(width.c_str());

	squareArea = lengthDbl * widthDbl;
	gallons = ceil(squareArea / feetPerGallons);

	cout << "You will need to purchase " << gallons << " gallons of paint to cover " << squareArea << " square feets";
}

/*TO DO: Check if input is valid*/
